domain/cloud/service/service_test.go

```diff
@@ -23,6 +25,10 @@ type serviceSuite struct {
 
 var _ = gc.Suite(&serviceSuite{})
 
+func dummyConfigSchemaProvider(ct string) (config.ConfigSchemaSource, error) {
+	return nil, fmt.Errorf("%w cloud %q schema provider", errors.NotFound, ct)
```

Comment:
shouldn't this be `fmt.Errorf("cloud %q schema provider %w", ct, errors.NotFound)`

can we name the function param `cloudType`

Reply:
yep